lib/puck/dependency_resolver.rb

```diff
@@ -41,7 +42,6 @@ def contained_bundler(gem_home, gemfile, lockfile, groups)
         begin
           line = __LINE__ + 1 # as __LINE__ represents next statement line i JRuby, and that becomes difficult to offset
           unit = scripting_container.parse(StringIO.new(<<-"EOS").to_inputstream, __FILE__, line)
-            require 'stringio'
```

Comment:
Actually, this line isn't needed anymore. Once it was used inside the scripting container to avoid errors being written to stderr. But the current implementation catches the errors, transformas them into "raw data" and rethrows them from outside the scripting container.

The outer require should be added as we have a direct dependency on `stringio`. Technically, it is only needed if bundler isn'n loaded, as it should be fulfilled by the following dependency chain `bundler/source_list` -> `bundler/source/rubygems` -> `net/http` -> `stringio`, which is autoloaded whenever a bundler definition is parsed. I'm not sure if we could easily (or if we even want to) run the specs without loading bundler.

Assuming that this is the explanation, it should be able to run `bundle exec puck` instead of just `puck` as a workaround (or adding the missing require as done in this PR).